src/mlpack/methods/reinforcement_learning/q_network.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file q_network.hpp
+ * @author Nishant Kumar
+ *
+ * This file is consists of the q network.
```

Comment:
```suggestion
 * This file consists of the q network.
```

Reply:
We generally use this statement, Definition or Implementation of Q-Network. Thanks 👍

Reply:
I would do the changes with the upcoming commit. Thanks. :+1: